tests/test_regressions.py

```diff
@@ -312,3 +312,9 @@ def test_issue207_runaway_format():
                            "          2 as two,",
                            "          3",
                            "   from dual) t0"])
+
+
+def token_next_doesnt_ignore_skip_cm():
```

Comment:
The prefix `test_` was missing on the test name, and wasn't actually running. Just corrected it, but wanted to give you an FYI :smile: